Source/Workers/decodeDraco.js

```diff
@@ -131,17 +126,15 @@ define([
                     quantization = {
                         quantizationBits : transform.quantization_bits(),
                         minValues : minValues,
-                        range : transform.range(),
-                        octEncoded : false
+                        range : transform.range()
                     };
                 }
                 draco.destroy(transform);
 
                 transform = new draco.AttributeOctahedronTransform();
                 if (transform.InitFromAttribute(attribute)) {
                     quantization = {
-                        quantizationBits : transform.quantization_bits(),
-                        octEncoded : true
```

Comment:
`ModelUtility.modifyShaderForQuantizedAttributes` will need to be updated accordingly.

Reply:
This also explains the test failures. I'll add `octEncoded` back in.